setup.py

```diff
@@ -27,6 +27,7 @@
     "fsspec >= 0.6.0",
     "toolz >= 0.8.2",
     "partd >= 0.3.10",
+    "importlib_metadata >= 3.6.0",
```

Comment:
OK - I omitted it originally because `importlib_metadata` is not just backwards compatability like `importlib_resources`, but it seems stable enough on closer inspection.